app/src/main/res/values/strings.xml

```diff
@@ -430,8 +430,16 @@ work. If not, see <https://creativecommons.org/publicdomain/zero/1.0/>.
     <string name="additional_contexts_and_projects">Additional contexts and projects</string>
     <string name="enter_space_seperated_contexts_and_projects_these_will">Enter space-separated @contexts and +projects. These will always be visible at todo.txt insert dialogs.</string>
     <string name="infinite">Infinite</string>
-    <string name="search_project">Search projects</string>
-    <string name="search_context">Search contexts</string>
+    <string name="filter_by_project">Filter by project</string>
+    <string name="filter_by_context">Filter by context</string>
     <string name="insert_project">Insert projects</string>
     <string name="insert_context">Insert contexts</string>
+    <string name="filter_todo">Filter todo</string>
+    <string name="filter_by_priority">Filter by priority</string>
+    <string name="filter_by_due_date">Filter by due date</string>
+    <string name="due_overdue">Overdue</string>
```

Comment:
I'm not sure overall about the wording `Filter`, but thats a simple replacement change before merge.